src/clr-angular/forms/datepicker/date-input.spec.ts

```diff
@@ -118,6 +126,18 @@ export default function() {
           sub.unsubscribe();
         });
 
+        it('should refocus input after selecting a date for a11y', () => {
+          const input: HTMLInputElement = context.testElement.querySelector('input');
+          expect(document.activeElement).not.toBe(input);
+
+          ifOpenService.open = true;
+          context.detectChanges();
+          ifOpenService.open = false;
```

Comment:
The name of this test, with this code, made me realize we force focus on the input even if I close the dropdown **without** selecting a date. It might create some weird behavior with someone opening the dropdown, then changing their mind and clicking on a link or button somewhere else in the page. 

Is this something we care about (cc @mathisscott)? 

If it is, how hard would it be to tie the focus to actual date selection using `DateNavigationService.selectedDayChange`? After a quick look it seems easy enough, but I may be missing something.

Reply:
Agree with the UX concern. Updated to use `DateNavigationService.selectedDayChange` instead.